css/styles.css

```diff
@@ -0,0 +1,27 @@
+.carousel {
+    margin-bottom: 60px;
+}
+
+.carousel-caption {
+    z-index: 10;
+    padding-bottom: 60px;
+}
+
+.carousel-inner .item {
+    width: 100%;
+    height: 600px;
+    background-size: cover;
+    background-position: center;
+}
+
+.main_img1 {
+    background-image: url(../img/wood-2.jpg);
```

Comment:
your `background-image` needs quotes surrounding URL. as so: url('../img/example.jpg');

Reply:
Thank you! I will add them in. It does work without them but i'm sure quotes are safer.